### PR TITLE
Added support for tcpdump command attachment

### PR DESCRIPTION
**Pull request by matsvanmolle**

### CODE REVIEW

**Comment by apaolillo** [comment]
@matsvanmolle I believe tcpdump is not a command wrapper, as you cannot give it a specific command in its own command line. It is more of a CommandAttachment in benchkit methodology (something that you run in parallel with your actual benchmark command). Could you update your patch to comply to that?

**Comment by apaolillo** [comment]
Hi @matsvanmolle let me know once this is ready. You can also remove the "draft" status once you consider it is ready to be reviewed and integrated. Thanks again.

**Comment by apaolillo** [comment]
@matsvanmolle are you still considering this one?

**Comment by matsvanmolle** [comment]
> @matsvanmolle are you still considering this one?

Yes, I will test this one in the coming week. Promised

**Comment by matsvanmolle** [comment]
@apaolillo I tested it and did some cleanup. However, I would like to point out that the password for sudo for tcpdump should be turned off.

File: benchkit/commandattachments/tcpdump.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        read_from_file: str = "",
+        show_timestamp: bool = False,
+        verbose_output: bool = False,
+        write_to_file: str = "",
+        snapshot_length: int = 0,
+        no_promiscuous_mode: bool = False,
+        display_packet_data: bool = False,
+        platform: Platform = None,
+    ) -> None:
+        self.interface = (interface,)
```
Comment (apaolillo) [inline benchkit/commandattachments/tcpdump.py:28]: this line is causing trouble, what are you trying to do?

Comment (matsvanmolle) [reply benchkit/commandattachments/tcpdump.py]: I think I did it in case multiple interfaces should be captured. I removed it.

**Review by apaolillo — APPROVED** [review]
well done Mats!

**Pull request merged**